src/assets/_project/_blocks/components/forms/_qg-forms.scss

```diff
@@ -153,10 +153,225 @@
   }
 
 }
-
-
 // hint text
 .hint{
   color: $qg-dark-gray-light;
   @include rem(font-size, 14px);
 }
+//------------------------------------------------------------------------------------
+// Updated form styles (migrated from forms.io repository to make all forms look the same)
+//------------------------------------------------------------------------------------
+
+.qg-forms-v2, #formio {
```

Comment:
i'm currently working on a ticket to support multiple forms in a page, so it would be nice to have the style selected by classname instead (ie. .formio_container) or wildcard id (ie. [id^=formio])
as `id` will need to be unique if there is multiple forms in a page (eg. #formio-xxxxx)

Reply:
my ticket: https://servicesmadesimpler.govnet.qld.gov.au/jira/browse/QOL-8650
style in demo: matrix asset in prod: style #249330
please let me know if there is any concern

Reply:
HI @devonpis,  Currently, this ID #formio is in use, so I included it so that we wouldn't break anything. If you are building a new component I would suggest using class .qg-forms-v2 and which can work with any number of forms and if we need any particular function we can include ID according to the requirements of a form.

Reply:
that's sounds reasonable, i'll wait for your merge and fully tested first and will implement the multiple form support later if it's neccessary, also i'll consider using .qg-forms-v2 as well